runtime/autoload/remote/host.vim

```diff
@@ -127,9 +158,25 @@ function! s:GetManifest() abort
 endfunction
 
 
+function! s:GetManifest() abort
+  let manifest = s:GetManifestPath()
+
+  if !filereadable(manifest)
+    " Check if an old manifest file exists and move it to the new location.
+    let old_manifest = s:GetOldManifestPath()
+    if filereadable(old_manifest)
+      call rename(old_manifest, manifest)
+    endif
+  endif
+
+  return manifest
+endfunction
+
+
 function! remote#host#LoadRemotePlugins() abort
-  if filereadable(s:GetManifest())
-    exe 'source '.s:GetManifest()
+  let manifest = s:GetManifest()
+  if filereadable(manifest)
+    exe 'source '.manifest
```

Comment:
should be 

    exe 'source '.shellescape(manifest)

to avoid problems like https://github.com/neovim/neovim/issues/5218.